GCMessengerSDKSample/GCMessengerSDKSample/AccountDetailsViewController.swift

```diff
@@ -91,14 +101,27 @@ class AccountDetailsViewController: UIViewController {
     }
     
     private func addCustomAttributes(account: MessengerAccount) {
```

Comment:
Can you pass the `customDict` into this method instead of converting it from a String to a map again?